apps/web-giddh/src/app/project-wise-accounting/revenue-expense-list/revenue-expense-list.component.ts

```
@@ -0,0 +1,506 @@
+     * @memberof RevenueExpenseListComponent
+     */
+    public searchAccount(query: string = '', page: number = 1): void {
+        if (this.accountSearchRequest.q !== query) {
+            this.accountSearchResponse = [];
+        }
+        this.accountSearchRequest.q = query;
+        this.accountSearchRequest.page = page;
+        this.accountSearchRequest.isLoading = true;
+
+        let requestObject = cloneDeep(this.accountSearchRequest);
+        delete requestObject.isLoading;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)